src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -84,6 +84,12 @@ export const ProjectCard = (props: ProjectCardProps) => {
     });
   };
 
+  const count = users.length;
+
+  const userList = count < 11 ? users : users.slice(0, 10);
```

Comment:
Theoretically, you could just use `userList = users.slice(0, 10);` here. (It won't complain if the list < 11). Not that it makes much of a difference though ;-)